packages/composition/dbaas/exoscale/redis.yml

```diff
@@ -0,0 +1,123 @@
+                  type: FromConnectionSecretKey
+                  name: REDIS_USERNAME
+                - fromConnectionSecretKey: REDIS_PASSWORD
+                  type: FromConnectionSecretKey
+                  name: REDIS_PASSWORD
+                - fromConnectionSecretKey: REDIS_URL
+                  type: FromConnectionSecretKey
+                  name: REDIS_URL
+                - fromConnectionSecretKey: ca.crt
+                  type: FromConnectionSecretKey
+                  name: ca.crt
+              patches:
+                - type: PatchSet
+                  patchSetName: annotations
+                - type: PatchSet
+                  patchSetName: labels
+                - type: FromCompositeFieldPath
+                  fromFieldPath: metadata.labels[crossplane.io/composite]
+                  toFieldPath: metadata.name
+                - type: FromCompositeFieldPath
+                  fromFieldPath: metadata.labels[crossplane.io/composite]
+                  toFieldPath: spec.writeConnectionSecretToRef.name
+                # Set redis settings
+                - type: FromCompositeFieldPath
+                  fromFieldPath: spec.parameters.service.redisSettings
+                  toFieldPath: spec.forProvider.redisSettings
+                # Set zone
+                - type: FromCompositeFieldPath
+                  fromFieldPath: spec.parameters.service.zone
+                  toFieldPath: spec.forProvider.zone
+                # Set IP filter
+                - type: FromCompositeFieldPath
+                  fromFieldPath: spec.parameters.network.ipFilter
+                  toFieldPath: spec.forProvider.ipFilter
+                # Set exoscale plan
+                - type: FromCompositeFieldPath
+                  fromFieldPath: spec.parameters.size.plan
+                  toFieldPath: spec.forProvider.size.plan
+                # Set maintenance day of week
+                - type: FromCompositeFieldPath
+                  fromFieldPath: spec.parameters.maintenance.dayOfWeek
+                  toFieldPath: spec.forProvider.maintenance.dayOfWeek
+                # Set maintenance timeOfDay
+                - type: FromCompositeFieldPath
+                  fromFieldPath: spec.parameters.maintenance.timeOfDay
+                  toFieldPath: spec.forProvider.maintenance.timeOfDay
+
+    additionalResources:
+      provider-exoscale-secrets-redis:
+        apiVersion: v1
+        kind: Namespace
+        metadata:
+          name: ${pkg.appcat.composition.dbaas.exoscale:secretNamespaces:redis}
+
+    secrets:
+      exoscale-dbaas-api-access:
+        metadata:
+          namespace: ${crossplane:namespace}
+        stringData:
+          EXOSCALE_API_KEY: '?{vaultkv:${cluster:tenant}/${cluster:name}/appcat/dbaas/provider-exoscale/access-key}'
+          EXOSCALE_API_SECRET: '?{vaultkv:${cluster:tenant}/${cluster:name}/appcat/dbaas/provider-exoscale/secret-key}'
+
+  crossplane:
```

Comment:
please remove the provider config, it will be managed in the provider itself, for more details check [this](https://github.com/vshn/component-appcat/pull/46).